humble/distribution.yaml

```diff
@@ -2161,6 +2161,25 @@ repositories:
       url: https://github.com/ros2/launch_ros.git
       version: humble
     status: maintained
+  ldrobot-lidar-ros2:
+    doc:
+      type: git
+      url: https://github.com/Myzhar/ldrobot-lidar-ros2.git
+      version: humble
+    release:
+      packages:
+      - ldlidar
```

Comment:
@Myzhar hmm, I see an MIT license mentioned in this package while everything points to Apache License, Version 2.0. Which one is it? Is it mixed?

Reply:
It was initially MIT, but then I changed to Apache-2. I search for remaining mentions to MIT

Reply:
Fixed